src/cascade/input_data/configuration/form.py

```diff
@@ -189,6 +189,7 @@ class Model(Form):
         default="iota_pos_rho_zero",
         display="(Advanced) Rate case",
     )
+    use_weighted_age_group_midpoints = OptionField([1, 0], default=1, constructor=int, nullable=True)
```

Comment:
OK, so this is an example of use.

Reply:
Oh, no. I should have moved that. It's in the wrong spot.